src/main/java/com/softdevsix/api/services/report/ReportService.java

```diff
@@ -1,15 +1,24 @@
 package com.softdevsix.api.services.report;
 
+import com.softdevsix.api.domain.entities.file.File;
+import com.softdevsix.api.domain.entities.project.Project;
 import com.softdevsix.api.domain.report.Report;
+import com.softdevsix.api.mappers.json.ProjectMapper;
+import com.softdevsix.api.repositories.IFileRepository;
+import com.softdevsix.api.repositories.IProjectRepository;
 import org.springframework.stereotype.Service;
 
 @Service
 public class ReportService implements IReportService {
 
     private final JsonReportReader reportReader;
+    private final IProjectRepository projectRepository;
+    private final IFileRepository fileRepository;
 
-    public ReportService(JsonReportReader reportReader) {
+    public ReportService(JsonReportReader reportReader, IProjectRepository projectRepository, IFileRepository fileRepository) {
```

Comment:
In my IDE I get this error, which occurs because you have more than one bean that implements the `IFileRepository` interface and Spring does not know which one to use to inject into the constructor of your ReportService class. Inside the constructor you could enter the suggestion that the IDE gives you.

![image](https://github.com/user-attachments/assets/d6d89064-5625-4e62-a3f9-8cbf1a10ad7e)

Reply:
Yes, I've added `FileMemoryRepository` as a `@Qualifier` parameter of the constructor. This will allow spring to realize which implementation should go there.
<!-- Reviewable comment -OD393mqEyf_rdXre907:b-6by7u9 -->
<!-- Sent from Reviewable.io -->